src/test/java/org/jenkinsci/plugins/scriptsecurity/sandbox/groovy/SecureGroovyScriptTest.java

```diff
@@ -842,4 +845,20 @@ public void testSandboxClassResolution() throws Exception {
             assertTrue(e.getMessage().contains("staticMethod java.lang.System gc"));
         }
     }
+    
+    /**
+     * Get the form button having the specified text/caption.
+     * @param htmlForm The form containing the button.
+     * @param caption The button text/caption being searched for.
+     * @return The button if found.
+     * @throws ElementNotFoundException Failed to find the button on the form.
+     */
```

Comment:
:ant: add a comment

```java
// TODO 1.627+ use standard version from HtmlFormUtil
```

so that we do not forget to clean up the temporary copy.